drivers/node/oracle/oracle.go

```diff
@@ -0,0 +1,126 @@
+package oracle
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/portworx/torpedo/drivers/node/ssh"
+
+	"github.com/libopenstorage/cloudops"
+	oracleOps "github.com/libopenstorage/cloudops/oracle"
+	"github.com/oracle/oci-go-sdk/v65/core"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// DriverName is the name of the aws driver
+	DriverName = "oracle"
+)
+
+type oracle struct {
+	ssh.SSH
+	ops               cloudops.Ops
+	instanceID        string
+	instanceGroupName string
+}
+
+func (o *oracle) String() string {
+	return DriverName
+}
+
+// Init initializes the node driver for oracle under the given scheduler
+func (o *oracle) Init(nodeOpts node.InitOptions) error {
+	instanceGroup := os.Getenv("INSTANCE_GROUP")
+	if len(instanceGroup) != 0 {
+		o.instanceGroupName = instanceGroup
```

Comment:
can you add the log initialization and use that instane for logging, you can refer https://github.com/portworx/torpedo/blob/master/drivers/node/ssh/ssh.go#L162

this is the new change introduced got torpedo logging

Reply:
Done!